### PR TITLE
Fix divrem for Z/nZ

### PR DESCRIPTION
**Pull request by thofma**

### CODE REVIEW

**Comment by thofma** [comment]
Does this make sense @fieker?

File: src/AlgAssAbsOrd/Conjugacy/Lifting.jl [hunk 1]
```diff
@@ -230,8 +235,9 @@ function Base.divrem(n::T, m::T) where T <: Union{nmod,Nemo.fmpz_mod}
       end
     end
   end
-  qq =  R(crt([x[2] for x = q], [x[1] for x = q])::fmpz)::T
+  qq = R(crt([x[2] for x = q], [x[1] for x = q])::fmpz)::T
   rr = n-qq*m
+  @show rr
```
Comment (tthsqe12) [inline src/AlgAssAbsOrd/Conjugacy/Lifting.jl:240]: you want to show this?

Comment (thofma) [inline src/AlgAssAbsOrd/Conjugacy/Lifting.jl:240]:
```suggestion
```

Comment (thofma) [reply src/AlgAssAbsOrd/Conjugacy/Lifting.jl]: You should use the suggestion feature more.

**Pull request merged**